rules/game_process.tex

```diff
@@ -31,14 +31,20 @@ \subsection{Robot States}
   \item[Unstiff.] It helps to facilitate a consistent and safe handling of the robots for remote competition.
     During any state, if all head buttons are pressed at least one second, the robot should move to a safe seated/crouched position and unstiffen all joints.
     So while in the \texttt{unstiff} state the robot is not allowed to move in any fashion! After booting, the robots are in their \texttt{unstiff} state.
-    Pressing the chest button once while in the \texttt{unstiff} state, permits the robot to stiffen its joints and return to the \texttt{initial} state, or a state as indicated by GameController.
+    Pressing the chest button once while in the \texttt{unstiff} state, permits the robot to stiffen its joints and return to the state indicated by GameController.
```

Comment:
It looks like you're assuming there will always be a game controller already running. This isn't usually the case, e.g. if you arrive at a field before GC is running or if you don't want to run GC (E.g. to do calibration). The transition from a button press in unstiff seems undefined when there is no GC. Can you define what state the robot should land in if there is no game controller running (I assume "setup").

Reply:
Yes, why not go into setup and then a second chest button press transitions to the penalized state (so no real change for the handling except the first reachable state via button interface is setup)

Reply:
I believe that the text still assumes that the transition to calibration takes place from initial (which can only be reached via GC). Truthfully, I would have dealt with all this differently by leaving the initial state the way it originally was and inserting a new state between initial and ready to accept the ref signal - this would have resulted in fewest changes to robot code. If you want to continue in your current direction with the setup state, then I think you will also need to enable a transition from the setup state to calibration rather than from initial to calibration.

Reply:
Having the new state before or after initial is just a matter of label as at the end the definition will be the same. To my opinion we should keep the current direction. At least I will update with this direction until we decided otherwise.